packages/sitecore-jss-vue/src/components/Text.ts

```diff
@@ -3,7 +3,7 @@ import { CreateElement, FunctionalComponentOptions, RenderContext } from 'vue';
 export interface TextProps {
```

Comment:
Let's add a unit test here? "should render number value"

Reply:
Yep, we have tests for number value: https://github.com/Sitecore/jss/blob/bug/479677/packages/sitecore-jss-vue/src/components/Text.test.ts#L93

Reply:
Wow, sorry :D 
I missed